deploy/kubernetes/base/controller.yaml

```diff
@@ -14,6 +14,7 @@ spec:
         app: gcp-compute-persistent-disk-csi-driver
     spec:
       serviceAccountName: csi-controller-sa
+      priorityClassName: gce-pd-csi-driver-controller
```

Comment:
add to readme

Reply:
I don't think the specific namespace name should be documented as it could become out of sync, but I'll mention the behavior of this priorityclass